reggie/configs/data/washington.yaml

```diff
@@ -8,10 +8,10 @@ fixed_width: false
 
 # -- VoteShield -- #
 
-date_format: '%m/%d/%Y'
+date_format: '%Y-%m-%d'
```

Comment:
You mentioned this, but the date fields in the db tables look empty, so probably this is the wrong format for the incoming data (or sometimes there are multiple different date formats incoming so you need a list here..)